Documentation/project-docs/dogfooding.md

```diff
@@ -161,7 +200,7 @@ as a reference to our nightly dotnet-core feed on myget:
 </configuration>
 
 ```
-Obviously **you need to update path in the XML to be the path to output directory for your build**.
+Obviously **you need to update the path in the XML to be the path to output directory for your build**.
 
 On Windows you also have the alternative of modifying the Nuget.Config
```

Comment:
This is now obsolete as you already pointed to it above. I would either merge both mentions of Nuget.config and put it in its own section or delete that one.